spyder/plugins/completion/kite/utils/tests/test_install.py

```diff
@@ -60,7 +60,8 @@ def finished():
                 FINISHED]
 
         # This status can be obtained the second time our tests are run
-        if not installation_statuses == ['Install finished']:
+        if installation_statuses not in [['Install finished'],
```

Comment:
You can remove `['Install finished']` from here because (as you correctly guessed), we changed that message to `Installation finished` but we forgot to update this test.